libraries/chain/db_maint.cpp

```diff
@@ -815,7 +816,7 @@ void database::process_bids( const asset_bitasset_data_object& bad )
    {
       const collateral_bid_object& bid = *itr;
       asset debt_in_bid = bid.inv_swan_price.quote;
-      if( debt_in_bid.amount > bdd.current_supply )
+      if( has_hf_20181128 && debt_in_bid.amount > bdd.current_supply )
```

Comment:
No need to add this check to another place?

Reply:
The change only has an effect in the selection of bids for the revival. The actual execution only uses the supply to cover anyway.
I did a snapshot comparison at block 22492000 to confirm.

Reply:
The change only has an effect in the selection of bids for the revival. The actual execution only uses the supply to cover anyway.
I did a snapshot comparison at block 22492000 to confirm.